Source/WebKit/WPEPlatform/wpe/drm/WPEDRMSeat.cpp

```diff
@@ -416,9 +417,9 @@ void Seat::handleKey(uint32_t time, uint32_t key, bool pressed, bool fromRepeat)
         g_source_attach(m_keyboard.repeat.source.get(), g_main_context_get_thread_default());
     }
 
-    // FIXME: make this configurable.
-    static const Seconds delay = 400_ms;
-    static const Seconds interval = 80_ms;
+    auto* settings = wpe_display_get_settings(wpe_view_get_display(wpe_event_get_view(event)));
```

Comment:
We don't need to get the view from the event in this case, we already have view variable.